examples/validating-public-input/aligned-integration/Cargo.toml

```diff
@@ -4,7 +4,7 @@ version = "0.1.0"
 edition = "2021"
 
 [dependencies]
-aligned-sdk = { git = "https://github.com/yetanotherco/aligned_layer", tag = "v0.6.0" }
+aligned-sdk = { git = "https://github.com/yetanotherco/aligned_layer", rev = "5ac0862ebace4e5971d1fd4c58f2c1973576d557" }
```

Comment:
should we leave this version commit?

Reply:
The version should be the current one. Should I point it to the current branch? If i point it to an existing release, it won't contain the changes made in this PR

Reply:
yes you are right. Let's make it point to the last commit of this branch, but have to remember to update it one we make a release

Reply:
Why is it not using the repository version ? I assume we may want to keep them working while operators upgrade maybe ?

Reply:
The actual content of the requests does not change so the examples should be usable even before the SDK is released, right?

Reply:
<img width="511" alt="image" src="https://github.com/user-attachments/assets/69fbc6db-7861-4ac3-9f1f-b715f4377fe6">